processes/community-membership.md

```diff
@@ -0,0 +1,182 @@
+    - Focus on holistic acceptance of contribution such as dependencies with other features, backwards / forwards compatibility, API and flag definitions, etc
+- Expected to be responsive to review requests
+- Mentor contributors and reviewers
+- May approve and merge code contributions for acceptance
+- Inactivity (such as for three months) may result in a review for suspension of all privileges until active again
+
+## Lead
+Subproject leads are the technical authority for a subproject in the Carvel project. They MUST have demonstrated both good judgment and responsibility towards the health of that subproject. Subproject leads MUST set technical direction and make or approve design decisions for their subproject - either directly or through delegation of these responsibilities.
+
+**Defined by**: Leads entry in the MAINTAINERS file
+
+### Requirements
+Unlike the roles outlined above, the Leads of a subproject are typically limited to a relatively small group of decision makers and updated as fits the needs of the subproject.
+
+The following apply to the subproject for which one would be a lead.
+- Deep understanding of the technical goals and direction of the subproject
+- Deep understanding of the domains of the subproject
+- Deep understanding of the design of the subproject
+- Sustained contributions to design and direction by doing all of:
+    - Authoring and reviewing proposals
+    - Initiating, contributing and resolving discussions (emails, GitHub issues, meetings)
+    - Identifying subtle or complex issues in designs and implementation PRs
+- Directly contributed to the subproject through implementation and / or review
+- Sponsored by a subproject lead
+    - With no objections from other maintainers
+    - Done through PR to update the MAINTAINERS file.
+        - Ensure your sponsors are @mentioned on the issue.
+        - Complete every item on the checklist (preview the current version of the template)
+        - Make sure that the list of contributions included is representative of your work on the project.
+        - Have your sponsoring reviewers reply confirmation of sponsorship: `+1`
+        - Once your sponsors have responded, your request will be reviewed by the project leads. Any missing information will be requested.
+
+### Responsibilities and privileges
+The following apply to the subproject for which one would be a lead.
+- Make and approve technical design decisions for the subproject.
+- Set technical direction and priorities for the subproject.
+- Define milestones and releases.
+    - Decides on when PRs are merged to control the release scope.
+- Mentor and guide approvers, reviewers, and contributors to the subproject.
+    - Ensure continued health of subproject
+    - Adequate test coverage to confidently release
+- Tests are passing reliably (i.e. not flaky) and are fixed when they fail
+- Ensure a transparent and healthy process for discussion and decision making is in place.
```

Comment:
@microwavables This item is something we'd like to get your thoughts on. Does it make sense to place the responsibility of "ensuring a transparent and healthy process" on a lead? Or would it be fairer to only expect them to "follow a transparent and healthy process"? The difference is who is expected to perform the enforcement/ensuring of a process.

Reply:
I think the whole team bears this responsibility while the community manager works to enforce it / remind the team. But, I do like the idea of the leads also being responsible for making sure the others on the team follow a transparent / healthy process.